### PR TITLE
feat: allow users to disable schedule on deploy [gh-779]

### PR DESCRIPTION
**Pull request by umutuzgur**
I hereby confirm that I followed the code guidelines found at [engineering guidelines](https://www.notion.so/checkly/Engineering-Guidelines-a3a165a203a04dc1a112f0e26b2f2d3f)

## Affected Components
* [x] CLI
* [ ] Create CLI
* [ ] Test
* [ ] Docs
* [ ] Examples
* [ ] Other

## Notes for the Reviewer

Allow users to disable schedule on deploy
> Resolves (after release) #779

### CODE REVIEW

File: packages/cli/src/commands/deploy.ts [hunk 1]
```diff
@@ -40,6 +40,10 @@ export default class Deploy extends AuthCommand {
       description: 'Shows the changes made after the deploy command.',
       default: false,
     }),
+    'schedule-on-deploy': Flags.boolean({
+      description: 'Disables automatic check scheduling after a deploy.',
```
Comment (nahuelon) [inline packages/cli/src/commands/deploy.ts:44]: Shouldn't be:
```suggestion
      description: 'Enables automatic check scheduling after a deploy.',
```

Comment (umutuzgur) [reply packages/cli/src/commands/deploy.ts:44]: That is correct. Good one 👍

**Review by nahuelon — DISMISSED** [review]
LGTM.

**Review by nahuelon — APPROVED** [review]
Nice deploy test fix!. LGTM.

**Pull request merged**